md/fault-tolerance.md

```diff
@@ -59,16 +62,21 @@ Also, in cluster environment, only one node is responsible to run the recovery a
 Configuration for the recovery is available in the `bonita-tenant-comunity-custom.properties` and can be updated using the
 [setup tool](BonitaBPM_platform_setup.md)
 
-```properties
+The default values of those properties should work for everyone. If the recovery task takes more than a few minutes, 
+you might want to change these values to run the recovery less often. Take a look at the metrics section to understand how
+to measure that.
 
-# configuration of the recovery mechanism (how works are recreated if lost)
+```properties
+# Recovery Mechanism: recreate works when they are lost due to incidents
+# All following configuration should work for everyone, it can be changed only to do performance tuning in limit-cases
 # Avoid verifying elements recently modified, by default no elements updated during the last hour is considered (ISO-8601 duration format).
 bonita.tenant.recover.consider_elements_older_than=PT1H
 # Duration after the end of the previous execution before a new one is started. By default recovery runs every 2 hours (ISO-8601 duration format)
 bonita.tenant.recover.delay_between_recovery=PT2H
-
 ```
 
+`bonita.tenant.recover.delay_between_recovery` is also the delay before the first recovery after the startup.
```

Comment:
Does it mean the recovery does not start right away at platform startup?
2 hours can seem a lot then !

Reply:
We still restart elements at startup. It is not considered as the recovery mechanism.

As explained here: the recovery mechanism `recreate works when they are lost due to incidents`

That means we wait 2 hours before checking for works lost due to an incident .. that happening since the startup.

Reply:
To be more clear: 
* We restart all elements at startup, regardless of their "age"
* 2 hours later and then every 2 hours, we run the recovery mechanism that recover elements only "older" than 1 hour

Reply:
> To be more clear:
> 
> * We restart all elements at startup, regardless of their "age"
> * 2 hours later and then every 2 hours, we run the recovery mechanism that recover elements only "older" than 1 hour

This part is much clearer that way. Can we include it in the doc like that? 👍🏼

Reply:
I've updated that is the description of the recovery mechanism